tests/php/Control/RequestHandlingTest.php

```diff
@@ -17,7 +17,7 @@
 use SilverStripe\Security\SecurityToken;
 
 /**
- * Tests for RequestHandler and HTTPRequest.
+ * Tests for functionality related to handling requests - not unit tests for RequestHandler.
```

Comment:
Modified this note because I was originally going to add the link tests in here, but then I noticed nothing in here is actually directly testing `RequestHandler` despite its name kiiinda suggesting that's what this class would be.